src/Search/Api/Elasticsearch/ElasticQueryBuilder.php

```diff
@@ -147,4 +148,17 @@ public function getQuery(): QueryExecutor
 
         return $exec;
     }
+
+    public function betweenDates(DateTimeImmutable $fromDate, DateTimeImmutable $toDate): QueryBuilder
+    {
+        $this->query('range', [
+            'sortDate' => [
+                'gte' => $fromDate->format('Y/m/d'),
```

Comment:
On the way in it gets stored as a long:
`[ 2017-02-03T10:27:54Z ] -> [ elasticsearch ] -> [Long(1486117674)]`

so anything we query and on the way out again it gets formatted back